webui/react/src/pages/ExperimentDetails/ExperimentTrials.tsx

```diff
@@ -454,10 +446,10 @@ const ExperimentTrials: React.FC<Props> = ({ experiment }: Props) => {
         show={showCheckpoint}
         title={`Best Checkpoint for Trial ${activeCheckpoint.trialId}`}
         onHide={handleCheckpointDismiss} />}
-      {showCompareTrials && selectedTrialsLoaded &&
+      {showCompareTrials &&
       <TrialsComparisonModal
         experiment={experiment}
-        trials={selectedTrials}
+        trials={selectedRowKeys}
```

Comment:
**nit:** `trialIds` to be more explicit that it's just the trial ids being passed in and not trial data.